core/src/main/java/nu/yona/server/crypto/seckey/SecretKeyUtil.java

```diff
@@ -124,6 +134,15 @@ public static LocalDateTime decryptDateTime(byte[] ciphertext)
 		return LocalDateTime.parse(new String(decryptBytes(ciphertext), StandardCharsets.UTF_8));
 	}
 
+	public static LocalDate decryptDate(byte[] ciphertext)
+	{
+		if (ciphertext == null)
+		{
+			return null;
+		}
+		return LocalDate.parse(new String(decryptBytes(ciphertext), StandardCharsets.UTF_8));
```

Comment:
There is no format specified for parsing. Isn't it better to do so?

Reply:
Makes sense. I'll do it for ``LocalDateTime`` too.